Makefile

```diff
@@ -265,12 +265,12 @@ else
 endif
 
 ifneq ($(filter y,$(DEP_FONT)),)
-	EXE_LIBS += $(QB_FONT_LIB)
-	CXXFLAGS += -DDEPENDENCY_LOADFONT
+	EXE_LIBS += $(FREETYPE_OBJS) $(FONT_OBJS)
 	QBLIB_NAME := $(addsuffix 1,$(QBLIB_NAME))
```

Comment:
Since the define is gone we can get rid of the QBLIB_NAME addition as well :) Very good stuff, it puts us one step closer to not having any more conditional compilation for `libqb.cpp`.

Reply:
Noted.